mne/io/besa/besa.py

```diff
@@ -86,13 +86,13 @@ def _read_evoked_besa_avr(fname, verbose):
     if 'Npts' in fields:
         fields['Npts'] = int(fields['Npts'])
         if fields['Npts'] != data.shape[1]:
-            logger.warn(f'The size of the data matrix ({data.shape}) does not '
-                        f'match the "Npts" field ({fields["Npts"]}).')
+            logger.warning(f'The size of the data matrix ({data.shape}) does '
+                           f'not match the "Npts" field ({fields["Npts"]}).')
```

Comment:
We shouldn't use `logger.warning` but rather our own `warn` from `mne.utils`. It does some magic based on the logging level and also sets `stacklevel` etc.

Reply:
I copied `logger.warning` from other parts of the code base. Should I change to `warn` everywhere?

Reply:
Yes I think so

Reply:
Done – except in a few places where comments state it shouldn't be changed.